.golangci.yml

```diff
@@ -19,6 +19,10 @@ linters:
     - revive
     - gofumpt
     - depguard
+    - nakedret
```

Comment:
Does anything speak against using `unimport` and `prealloc` as well (both from the same author)?
Because especially `prealloc` could be nice to have...

Reply:
unimport isn't in golangci-lint afaik
prealloc can be useful yes, a test shows 25 hits.